docs/website/optimizations.md

```diff
@@ -380,3 +380,71 @@ Xamarin.Mac supports loading assemblies dynamically, and those assemblies
 might not have been known at build time (and thus not optimized).
 
 The default behavior can be overridden by passing `--optimize=-register-protocols` to mtouch/mmp.
+
+Remove the dynamic registrar
+----------------------------
+
+Both Xamarin.iOS and Xamarin.Mac include support for [registering managed
+types][1] with the Objective-C runtime. It can either be done at build time or
```

Comment:
^ I don't think that syntax works with mark-down (on the website), you have a inline the URL within a `()`